aiosmtpd/main.py

```diff
@@ -224,12 +233,13 @@ def main(args: Optional[Sequence[str]] = None) -> None:
                 file=sys.stderr,
             )
             sys.exit(1)
-        nobody = pwd.getpwnam("nobody").pw_uid
+        suid_user_id = pwd.getpwnam(args.suid_user).pw_uid
         try:
-            os.setuid(nobody)
+            os.setuid(suid_user_id)
         except PermissionError:
             print(
-                'Cannot setuid "nobody"; try running with -n option.', file=sys.stderr
+                f'Cannot setuid to "{args.suid_user}"; try running with -n option.',
+                file=sys.stderr,
             )
             sys.exit(1)
```

Comment:
**suggestion (non-blocking)** Since we're going ahead and exiting with status 1 we *could* go ahead and simply `sys.exit(f"Cannot ...")` -- and use the (documented) functionality of sys.exit. However one of our tests *does* check that the exception `.value.code == 1`, and it would have to equal the error message instead. 

Not a big deal, but that would simplify things here just a bit.

Reply:
This feels very tangential to this PR.  Personally, I feel like the explicit print and exit code is more readable -- I've never seen the `sys.exit("...")` formulation, though it's by no means new!

So I'm not going to include this change in the PR, because I'm not entirely sold on it.